spec/like_spec.rb

```diff
@@ -0,0 +1,15 @@
+require 'rails_helper'
+
+RSpec.describe Like, type: :model do
+  let!(:user) { User.create!(name: 'Test User', posts_counter: 0) }
+  let!(:post) { Post.create!(title: 'Test Post', author: user, comments_counter: 0, likes_counter: 0) }
+
+  it 'increments the likes counter when created' do
+    expect { Like.create(user:, post:) }.to change { post.reload.likes_counter }.by(1)
+  end
+
+  it 'decrements the likes counter when destroyed' do
+    like = Like.create(user:, post:)
+    expect { like.destroy }.to change { post.reload.likes_counter }.by(-1)
+  end
```

Comment:
- [ ] Any custom method in your models should have corresponding tests. For example, the `update_likes_counter` method in your Post model should be tested

Reply:
![image](https://github.com/lerfast/blog-app/assets/111823606/02ed8dbb-7006-421a-b335-45d955e8ffa6)
Did it!!!